### PR TITLE
microsite: Add blog entry about LF course

### PR DESCRIPTION
**Pull request by jorgelainfiesta**
Signed-off-by: Jorge Lainfiesta <jorge@roadie.io>

## Hey, I just made a Pull Request!

As introduced in a Slack conversation, the Linux Foundation published a new course on Backstage and it might be a good idea to increase awareness of this new resource by publishing it in the blog. 

The text pretty much paraphrases the original blog post from the Linux Foundation, hence the tone. I'm happy to adjust the style or anything else if needed.

Thanks!

## Screenshot

Screenshot 2022-11-15 at 14 17 58

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: microsite/blog/2022-11-15-linux-foundation-introduction-to-backstage-course.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+description: The new Linux Foundation course is available for free on edX and is designed to help individuals understand how to map Backstage to their organization needs
+author: Jorge Lainfiesta, Roadie
+authorURL: https://www.linkedin.com/in/jrlainfiesta/
+---
+
+Backstage continues to grow in [popularity and maturity](https://roadie.io/blog/backstage-consolidating-its-role-in-the-cloud-native-ecosystem/), with industry leaders not only adopting the framework but actively participating in the community through contributions and commercial offerings.  One of the most recent contributions comes from the Linux Foundation, [launching an introductory course](https://training.linuxfoundation.org/blog/23107/) aimed at DevOps engineers or professionals working on Developer Productivity or Developer Experience. The course is available for free on edX, with a paid option for the certified track. 
+
+[![Introduction to Backstage: Developer Portals Made Easy (LFS142x): Enroll!](assets/22-11-15/Introduction-to-Backstage-Developer-Portals-Made-Easy-2-768x432.png)](https://www.edx.org/course/introduction-to-backstage-developer-portals-made-easy)
+
```
Comment (Rugvip) [inline microsite/blog/2022-11-15-linux-foundation-introduction-to-backstage-course.md:11]: Needs a `<!--truncate-->` here to create the fold, see other blogposts for an example

**Review by Rugvip — COMMENTED** [review]
🎉 leaving review as assigned, but just wanted to chime in to make sure this is covered :grin:

File: microsite/blog/2022-11-15-linux-foundation-introduction-to-backstage-course.md (hunk 1 above)
Comment (jorgelainfiesta) [reply microsite/blog/2022-11-15-linux-foundation-introduction-to-backstage-course.md:11]: Thanks! I only saw the last entry and it didn't have that. Will add it now!

File: microsite/blog/2022-11-15-linux-foundation-introduction-to-backstage-course.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+<!--truncate-->
+
+The course, [Introduction to Backstage: Developer Portals Made Easy (LFS142x)](https://www.edx.org/course/introduction-to-backstage-developer-portals-made-easy), starts discussing the benefits of adopting a Developer Portal. Then, it dives into what is Backstage, its main features (Catalog, Scaffolder, TechDocs), and how to map its capabilities to your organization. At last, it provides tips on how to connect with the community to ease up your adoption journey.
+
+The course author, [Jorge Lainfiesta](https://www.linkedin.com/in/jrlainfiesta/), is a Technical Marketing Manager at [Roadie](https://roadie.io). Jorge has a background in software engineering (PayPal) and digital communication (UCLA). He’s been working around Backstage since it was open sourced by Spotify and co-hosts community initiatives like the Backstage Users Unconference.
```
Comment (jhaals) [inline microsite/blog/2022-11-15-linux-foundation-introduction-to-backstage-course.md:16]: We also need to place Lainfiesta into our vale accept.txt file to please or spell checker :) https://github.com/backstage/backstage/blob/master/.github/vale/Vocab/Backstage/accept.txt

**Review by jhaals — CHANGES_REQUESTED** [review]
Nice work! We also need your surname in the vocab file 😅

File: microsite/blog/2022-11-15-linux-foundation-introduction-to-backstage-course.md (hunk 2 above)
Comment (jorgelainfiesta) [reply microsite/blog/2022-11-15-linux-foundation-introduction-to-backstage-course.md:16]: Ohh gotcha! I added it now! Thanks!

**Review by jhaals — APPROVED** [review]

**Comment by jhaals** [comment]
Thanks!

**Pull request merged**